testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/security/RunAsPrincipalTestCase.java

```diff
@@ -218,4 +232,40 @@ private Throwable checkEjbException(Throwable ex) {
         }
         return ex;
     }
+
+    private boolean logContainsWARN() {
+        String jbossHome = System.getProperty("jboss.inst");
+        String logPath = String.format("%s%sstandalone%slog%sserver.log", jbossHome,
+                (jbossHome.endsWith(File.separator) || jbossHome.endsWith("/")) ? "" : File.separator,
+                File.separator, File.separator);
+        logPath = logPath.replace('/', File.separatorChar);
+        BufferedReader fileReader = null;
+        try {
+            fileReader = Files.newBufferedReader(Paths.get(logPath)); // UTF8 is used by default
+        } catch (MalformedInputException e1) {
+            // some windows machines could accept only StandardCharsets.ISO_8859_1 encoding
+            try {
+                fileReader = Files.newBufferedReader(Paths.get(logPath), StandardCharsets.ISO_8859_1);
```

Comment:
I'm not too sure this is really requires as UTF-8 should work. Either way though we need to ensure the `fileReader` is closed at the end.